lib/vagrant-orchestrate/command/init.rb

```diff
@@ -126,34 +126,39 @@ def execute
           options[:ssh_username] ||= DEFAULT_SSH_USERNAME
           options[:ssh_private_key_path] ||= DEFAULT_SSH_PRIVATE_KEY_PATH unless options[:ssh_password]
 
+          write_vagrant_files(options)
+          @env.ui.info(I18n.t("vagrant.commands.init.success"), prefix: false)
+
+          # Success, exit status 0
+          0
+        end
+        # rubocop:enable Metrics/AbcSize, MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
+        private
+
+        def write_vagrant_files(options)
           contents = TemplateRenderer.render(Orchestrate.source_root.join("templates/vagrant/Vagrantfile"),
```

Comment:
This is just rubocop suggested formatting changes. The real meat is the removal of the `plugins: options[:plugins],` from old like 143.